packages/okta-react/src/SecureRoute.js

```diff
@@ -10,19 +10,20 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useOktaAuth } from './OktaContext';
-import { useHistory, Route } from 'react-router-dom';
+import { Route } from 'react-router-dom';
```

Comment:
Where do you use Route?

Reply:
Forgot to move the history logic back, good catch, Thanks!